### DISCUSSION TITLE
Bug - script can no longer delete messages after first 25

### DISCUSSION THREAD

**Discussion by JulioCMendo** [victornpb/undiscord#298] [1. Questions & Answers] [answered]
Hello. On every chat, the script is no longer able to find messages past the first 25 automatically. It seems to be related to the pagination or something, I'm unsure.
![image](https://user-images.githubusercontent.com/33879918/147993809-9a7e37d1-486e-4f09-8624-f91c84a3c08a.png)

**Comment by SakiSakiSakiSakiSaki** [comment]
Same.

**Comment by Kaustubhjoshi-debug** [comment]
Same

**Comment by ghost** [comment, marked as answer]
https://discordstatus.com/incidents/7ztcrjdtcx3v

**Reply by JulioCMendo** [reply]
Oh, I thought that since the problem was occurring with older messages, this had nothing to do with it.

**Reply by Solomon646** [reply]
Thank you.

**Comment by gqqjhffr** [comment]
This is still happening for me.

**Reply by Solomon646** [reply]
Same.

**Reply by Toby222** [reply]
Please check if it's Discord's fault first.
You can see on https://discordstatus.com/ (specifically [here](https://discordstatus.com/incidents/cbrtv3whq4x8)) that search was down

**Comment by andrewkm** [comment]
Looks like this is suddenly happening again.
Wasn't yesterday. RIP.

**Comment by killerszsdf** [comment]
Its still happening. March 4th

Edit: And I do see the "resolve" on discord status but it isn't resolved

**Comment by aijorgenson** [comment]
https://github.com/victornpb/deleteDiscordMessages/pull/323

**Comment by twardy95** [comment]
Still happening

**Comment by AverageHoarder** [comment]
Happening on my end as well. Both in Firefox and Chrome and with both undiscord 5.0.3 by victornpb and undiscord 5.0.5 edited by Ashintosh.
Is there a hacky quick fix? I'd rather not delete 25k+ messages in 25 message bites.

NVM it appears to be working again.

**Comment by JulioCMendo** [comment]
Please, if possible, close this thread, since this only happens when discord service is down in some capacity, there's no need for this to be still up @victornpb

**Comment by victornpb** [comment]
Please,

Check https://discordstatus.com/ to see if **Discord** (not **Undiscord**) is having issues, specially the **API** and **Search** services.

If that's the case we can't do anything other than wait services to normalize.
